bmrg-flow/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: bmrg-flow
 description: Boomerang Flow
 type: application
-version: 4.11.3
+version: 4.11.4
 appVersion: 3.5.0
```

Comment:
Will need to update this to 3.5.1 or 3.6.0 depending on when its going out officially.

Reply:
Sure, I can update appVersion to 3.5.1